src/main/java/com/conveyal/gtfs/validator/PostLoadValidator.java

```diff
@@ -46,6 +49,20 @@ private void validateCalendarServices () {
         }
     }
 
+    /**
+     * Validate that stops are not in locations with no people. This can happen from incorrect coordinate transforms
+     * into WGS84. Stops are often located on "null island" at (0,0). This can also happen in other coordinate systems
+     * before they are transformed to WGS84: the origin of a common French coordinate system is in the Sahara.
+     */
+    private void validateStopPopulationDensity () {
+        BooleanAsciiGrid popGrid = BooleanAsciiGrid.forEarthPopulation();
```

Comment:
Is this population grid loaded and used anywhere else? I'm wondering if we should be concerned about resource management or additional time delays here.

Reply:
No, it was only used by a GTFS validator that was not included in r5 when we moved the gtfs-lib code in, so is not currently used elsewhere. This is a 4.3 kb text file containing ones and zeros which gets loaded into a BitSet, so should amount to a few hundred bytes in memory. There would be some cost of reading that file and creating that object, but this is in the context of loading and validating GTFS which involves loading large volumes of numbers from text files into a database. So I'd say it's a one-off cost per feed that's a similar action to what GTFS ingestion already requires, but several orders of magnitude smaller.  In principle a single instance of the grid could be cached, but at a few hundred bytes it's so tiny that I don't think it will have any impact.